server/mdm/apple/apple_mdm.go

```diff
@@ -49,6 +49,9 @@ const (
 	// FleetPayloadIdentifier is the value for the "<key>PayloadIdentifier</key>"
 	// used by Fleet MDM on the enrollment profile.
 	FleetPayloadIdentifier = "com.fleetdm.fleet.mdm.apple"
+
+	// SCEPProxyPath is the HTTP path that serves the SCEP proxy service. The path is followed by identifier.
+	SCEPProxyPath = "/mdm/scep/proxy/"
```

Comment:
not part of this PR, just a heads-up: does this path needs to be added here? https://fleetdm.com/guides/what-api-endpoints-to-expose-to-the-public-internet

Reply:
Yes, made a note.